guides/v2.3/graphql/queries/cateogry-list.md

```diff
@@ -0,0 +1,366 @@
+                "path": "1/2/20/21/23",
+                "url_path": "women/tops-women/jackets-women",
+                "url_key": "jackets-women"
+              },
+              {
+                "id": 24,
+                "level": 4,
+                "name": "Hoodies & Sweatshirts",
+                "path": "1/2/20/21/24",
+                "url_path": "women/tops-women/hoodies-and-sweatshirts-women",
+                "url_key": "hoodies-and-sweatshirts-women"
+              },
+              {
+                "id": 25,
+                "level": 4,
+                "name": "Tees",
+                "path": "1/2/20/21/25",
+                "url_path": "women/tops-women/tees-women",
+                "url_key": "tees-women"
+              },
+              {
+                "id": 26,
+                "level": 4,
+                "name": "Bras & Tanks",
+                "path": "1/2/20/21/26",
+                "url_path": "women/tops-women/tanks-women",
+                "url_key": "tanks-women"
+              }
+            ]
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+### Return breadcrumb information
+
+The following query returns breadcrumb information about the women's `Tops` category (`id` = 25).
+
+**Request**
+
+```graphql
+{
+  categoryList(filters: {ids: {eq: "25"}}) {
+    id
+    level
+    name
+    breadcrumbs {
+      category_id
+      category_name
+      category_level
+      category_url_key
+    }
+  }
+}
+```
+
+**Response**
+
+```json
+{
+  "data": {
+    "categoryList": [
+      {
+        "id": 25,
+        "level": 4,
+        "name": "Tees",
+        "breadcrumbs": [
+          {
+            "category_id": 20,
+            "category_name": "Women",
+            "category_level": 2,
+            "category_url_key": "women"
+          },
+          {
+            "category_id": 21,
+            "category_name": "Tops",
+            "category_level": 3,
+            "category_url_key": "tops-women"
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+## Input attributes
+
+You must specify the `filters` attribute as input to your query.
+
+Attribute | Data type | Description
+--- | --- | ---
+`filters` | CategoryFilterInput | Contains filter definitions
+
+### CategoryFilterInput object
+
+The `CategoryFilterInput` object defines the filters to be used in this query.
+
+Attribute | Data type | Description
+--- | --- | ---
+`ids` | FilterEqualTypeInput | Filters by the specified category IDs
+`name` | FilterMatchTypeInput | Filters by the display name of the category
+`url_key` | FilterEqualTypeInput | Filters by the part of the URL that identifies the category
+
+### FilterEqualTypeInput object
+
+Use the `FilterEqualTypeInput` object to construct filters that search by category ID or URL key.
+
+Attribute | Data type | Description
+--- | --- | ---
+`eq` | String | Use this attribute to exactly match the specified string. For example, to filter on a specific URL key, specify a value like `shorts-women`
```

Comment:
Using a mix of terminal punctuation for these descriptions seems like an anti-pattern to me...